<ui version="4.0" >
 <class>VerifyAccount</class>
 <widget class="QDialog" name="VerifyAccount" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="styleSheet" >
   <string>QDialog {
 
background-color:rgb(246,243,243);

}
</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>-50</x>
     <y>206</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>30</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>want to talk with you.</string>
   </property>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>60</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>replay</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RB_Accept" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string> acceptï¼Œset local name</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="RB_Refuse" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>160</y>
     <width>89</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>refuse</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CB_ADDBlack" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>180</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>refuse this gay contact with me</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Local_name" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>80</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_group" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>120</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>120</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>add to this group</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>10</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>hi, i am</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>10</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>fetion no:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_desc" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>10</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Label_FetionID" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VerifyAccount</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VerifyAccount</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
